vbfml/scripts/err_analysis

```diff
@@ -332,7 +334,7 @@ def average(ctx, input_dir: str, quantity: str, threshold: float) -> None:
     # Get the images for the events where quantity > threshold
     # and compute the average for these images.
     mask = df[quantity] > threshold
-    classes = ["ewk_17", "v_nlo_qcd_17"]
+    classes = ["bkg", "signal"]
```

Comment:
Thanks for fixing this, could we keep the `_17` labels to mark that these are only 2017 data? So labels can be `bkg_17` and `signal_17`.